### PR TITLE
Demos: improve accessibility

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: JSDemos/Demos/Common/ActionAndListsOverview/jQuery/index.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
         </div>
         <div class="tile">
           <script type="text/html" id="tile-template">
-            <img class="tile-image" src="../../../../images/hotels/{{ FileName }}" />
+            <img alt="{{FileName}}" class="tile-image" src="https://js.devexpress.com/Demos/WidgetsGallery/JSDemos/images/hotels/{{ FileName }}" />
```
Comment (AlexanderMoiseev) [inline JSDemos/Demos/Common/ActionAndListsOverview/jQuery/index.html]: i dont get it) why absolute path

Comment (marker-dao) [reply JSDemos/Demos/Common/ActionAndListsOverview/jQuery/index.html]: Replaced: [d223abbadd3d41efe5957c9016e5e242fc8c67e1].

**Review by AlexanderMoiseev — DISMISSED** [review]

File: JSDemos/Demos/CheckBox/Overview/AngularJS/index.js [hunk 2]
```diff
@@ -5,31 +5,52 @@ DemoApp.controller('DemoController', ($scope) => {
...
+      elementAttr: {
+        'aria-label': 'Disabled',
+      },
     },
     customSize: {
       value: null,
       iconSize: 30,
+      elementAttr: {
+        'aria-label': 'Custom size',
+      },
```
Comment (ksercs) [inline JSDemos/Demos/CheckBox/Overview/AngularJS/index.js:53]: We don't update AngularJS demos!
No need to retrieve this, but note for the future plz

File: JSDemos/Demos/CheckBox/Overview/React/App.js [hunk 3]
```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import { CheckBox } from 'devextreme-react/check-box';
 
+const checkedLabel = { 'aria-label': 'Checked' };
+const unCheckedLabel = { 'aria-label': 'Unchecked' };
```
Comment (ksercs) [inline JSDemos/Demos/CheckBox/Overview/React/App.js]: `un` is a prefix, not a separate word
```suggestion
const uncheckedLabel = { 'aria-label': 'Unchecked' };
```

File: JSDemos/Demos/HtmlEditor/ToolbarCustomization/Vue/App.vue [hunk 4]
```diff
@@ -8,8 +8,9 @@
         <DxItem name="redo"/>
         <DxItem name="separator"/>
         <DxItem
-          :accepted-values="headerValues"
           name="header"
+          :accepted-values="headerValues"
+          :option="headerOptions"
```
Comment (ksercs) [inline JSDemos/Demos/HtmlEditor/ToolbarCustomization/Vue/App.vue]: Shouldn't it be like this?
```suggestion
          :options="headerOptions"
```

File: NetCoreDemos/Views/Common/DialogsAndNotificationsOverview.cshtml [hunk 5]
```diff
@@ -9,14 +9,14 @@
...
-                <img src="@property.Image" />
+                <img alt="@property.Address" src="@property.Image" />
                 <div class="item-options">
                     <div>
                         <div class="address">@property.Address</div>
                         <div class="price large-text">$@string.Format("{0:n0}", property.Price)</div>
                         <div class="agent">
                             <div id="@("house" + property.ID)">
-                                <img src="~/images/icon-agent.svg">
+                                <img alt="Listing agent" src="~/Content/Images/icon-agent.svg">
```
Comment (ksercs) [inline NetCoreDemos/Views/Common/DialogsAndNotificationsOverview.cshtml]: Do we really need to change the path to svg here?

File: JSDemos/Demos/CheckBox/Overview/React/App.js (hunk 3 above)
Comment (marker-dao) [reply JSDemos/Demos/CheckBox/Overview/React/App.js]: Fixed: [e74e9aa0894fdc88f313789f8799b626ce473904].

File: NetCoreDemos/Views/Common/DialogsAndNotificationsOverview.cshtml (hunk 5 above)
Comment (marker-dao) [reply NetCoreDemos/Views/Common/DialogsAndNotificationsOverview.cshtml]: Fixed: [e74e9aa0894fdc88f313789f8799b626ce473904].

File: JSDemos/Demos/HtmlEditor/ToolbarCustomization/Vue/App.vue (hunk 4 above)
Comment (marker-dao) [reply JSDemos/Demos/HtmlEditor/ToolbarCustomization/Vue/App.vue]: Fixed: [e74e9aa0894fdc88f313789f8799b626ce473904].

**Review by ksercs — DISMISSED** [review]

**Review by ksercs — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Review by AlexanderMoiseev — APPROVED** [review]

**Review by alexanderPolosatov — APPROVED** [review]

**Pull request merged**